File: c_work/221226/ex05.c
#include <stdio.h>

int main(){

    int arr1[2][2] = {1,2,3,4};
    int arr2[3][2] = {5,6,7,8,9,10};
    int arr3[1][2] = {11,22};

    int (*ptr)[2] = arr1;
    printf("*(ptr[0]) %d ",*(ptr[0]));
    printf("ptr[0][1] %d\n",ptr[0][1]);
    printf("ptr[1][0] %d ",ptr[1][0]);
    printf("ptr[1][1] %d\n\n",ptr[1][1]);

    ptr = arr2;
    printf("ptr[0][0] %d ",ptr[0][0]);
    printf("ptr[0][1] %d\n",ptr[0][1]);
    printf("ptr[1][0] %d ",ptr[1][0]);
    printf("ptr[1][1] %d\n",ptr[1][1]);
    printf("ptr[2][0] %d ",ptr[2][0]);
    printf("ptr[2][1] %d\n\n",ptr[2][1]);

    ptr = arr3;
    printf("ptr[0][0] %d ",ptr[0][0]);
    printf("ptr[0][1] %d\n",ptr[0][1]);


    return 0;
}

File: data_cv2_work/230224/ex05.py
import numpy as np

arr1 = np.array([1,9,2,3,5])
arr1.sort()
print(arr1)

arr2 = np.array([1,9,2,3,5])
arr2.sort()
print(arr2[::-1])

# ??? 나중에 인터넷 쳐볼것
arr3 = np.array([[5,11,10,2,3],[4,1,10,22,3]])
arr3.sort(axis=0)
print(arr3)

arr4 = np.array([[5,11,10,2,3],[4,1,10,22,3]])
arr4.sort(axis=1)
print(arr4)